packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,223 @@
+    * **Phone Number**: A public phone number that customers can call for support
+
+* **Escalation support contact**: This is a private support contact for Okta to use when contacting your company about your integration. It isn't shared with customers. Okta needs to reach your company's escalation support team in an emergency, so ideally you need to provide a monitored email distribution list. You can also provide a phone number, but ensure that the phone number (or email) isn't a general support contact that's queued with regular enquiries.
+
+## Test account guidelines
+
+You need to create a test account for your app so that the OIN team can use it to test and verify your integration. The test account details and credentials are specified in the **General Settings** > **Test Account** fields from the OIN Manager.
+
+In general, the test account allows the OIN team to verify that your integration flow works as expected for your use case. The test account is typically an admin user in your app with additional privileges depending on your use case:
+
+* For a lifecycle management integration, ensure that your admin test account has HR admin privileges to onboard, change roles, or offboard employees on your app.
+* For an SSO or SCIM integration, ensure your admin test account has privileges to configure SSO and SCIM (even if your app is already configured with SSO or SCIM). The OIN team needs to verify whether users and/or groups were created by SCIM provisioning or by SAML/OIDC (JiT) on your application.
+* For an API service integration, ensure that your admin test account has privileges to configure an API integration and trigger API requests in your application.
+
+> **Note:** The OIN team recommends `isvtest@okta.com` as the test account username, however, you can provide an alternative username with a different domain.
+
+## Customer configuration document guidelines
+
+A configuration guide helps your customers understand how to configure your Okta integration to work with your cloud application.
+
+You need to provide a configuration guide as part of the OIN submission process. Your guide is provided to administrators through the Okta Admin Console. Okta checks your document for general adherence to the configuration guide instructions. After your integration is published to the OIN, you can make the link public or customer‐accessible.
+
+>**Note:** A configuration guide is required for SCIM and OIDC integrations. It's optional for SAML integrations, as Okta supplies its own documentation with the apps.
+
+You can create the guide in whatever format works best for you (a webpage, a Google doc, or a PDF are all acceptable), but you need to provide an accessible link to the doc for verification.
+
+### Configuration guide content
+
+The following are section suggestions for your configuration guide:
+
```

Comment:
It will probably look better as a table, but we loose the heading anchors in that format. This content existed for the last 2 years at least, so I'm trying not to change section headers too much.

Reply:
Ah, that makes more sense.